### ISSUE TITLE
[Bug]: app is opening but not responding

### ISSUE THREAD

**Issue by adithi1717** [TranslucentTB/TranslucentTB#707]
### Please check for a duplicate first

- [X] I have checked the issue tracker if my bug has already been fixed or answered in another (possibly closed) issue.

### Steps to reproduce the issue

hey so i've been using this app for the past 2+ years and only after the recent updates i've started facing problems. I have uninstalled and installed the app several times but everytime i open the app and put the required taskbar settings it doesn't apply to the taskbar. It remains the same. Sometimes when i open the app it immediately closes and quits. Please help me with the issue .

### Expected results

I want the app to run like it used to and not close the moment i run the app.

### Actual results

the app continues to not respond

### Windows version

Windows 11 Insider builds (builds 22623 and up). Please mention exact build.

**Comment by sylveon** [comment]
Your windows version says insider builds. Is that correct?

**Comment by dayumcodes** [comment]
will it not work in insider builds??

**Comment by bloodyeyesblackblood** [comment]
> will it not work in insider builds??

sadly it does not work with the insider builds since windows "did" something. #537 (also as i know sylveon is currently workin' on this ~ 💕)
 
also your problem sound a little different but ya :( 
a lot of things don't really work out on insider builds

**Comment by adithi1717** [comment]
Yes that's right

**Comment by sylveon** [comment]
Duplicate of #537
